workshops/ARC.md

```diff
@@ -44,3 +44,60 @@ for hacking.
 
 [portfolio]: portfolio/README.md
 [twilio]: twilio/README.md
+
+## Concept Arc
+
+- pair programming
```

Comment:
These all need to be capitalised correctly.

Also:

- `html`->`HTML`
- `dom`->`DOM`
- etc.